example-jobs/checkmarx-security-job-example.groovy

```diff
@@ -10,7 +10,7 @@ new CheckmarxSecurityJobBuilder(
         username: "sample-username",
         password: "sample-password",
         groupId: "",
-        scanRepo: "https://github.com/OrlandoSoto/ckan-browser-tests",
+        scanRepo:[[url: "https://github.com/OrlandoSoto/ckan-browser-tests"]],
```

Comment:
No, it is correct. The builder was changed to accept a list of repos to scan. See #51.